src/data/pokedex_orders.h

```diff
@@ -1,1609 +1,1609 @@
-    364,  // Sealeo
-    273,  // Seedot
-     86,  // Seel
-    161,  // Sentret
-    336,  // Seviper
-    319,  // Sharpedo
-    292,  // Shedinja
-    372,  // Shelgon
-     90,  // Shellder
-    275,  // Shiftry
-    285,  // Shroomish
-    213,  // Shuckle
-    353,  // Shuppet
-    266,  // Silcoon
-    227,  // Skarmory
-    188,  // Skiploom
-    300,  // Skitty
-    289,  // Slaking
-    287,  // Slakoth
-     80,  // Slowbro
-    199,  // Slowking
-     79,  // Slowpoke
-    218,  // Slugma
-    235,  // Smeargle
-    238,  // Smoochum
-    215,  // Sneasel
-    143,  // Snorlax
-    361,  // Snorunt
-    209,  // Snubbull
-    338,  // Solrock
-     21,  // Spearow
-    363,  // Spheal
-    167,  // Spinarak
-    327,  // Spinda
-    325,  // Spoink
-      7,  // Squirtle
-    234,  // Stantler
-    121,  // Starmie
-    120,  // Staryu
-    208,  // Steelix
-    185,  // Sudowoodo
-    245,  // Suicune
-    192,  // Sunflora
-    191,  // Sunkern
-    283,  // Surskit
-    333,  // Swablu
-    317,  // Swalot
-    260,  // Swampert
-    277,  // Swellow
-    220,  // Swinub
-    276,  // Taillow
-    114,  // Tangela
-    128,  // Tauros
-    216,  // Teddiursa
-     72,  // Tentacool
-     73,  // Tentacruel
-    175,  // Togepi
-    176,  // Togetic
-    255,  // Torchic
-    324,  // Torkoal
-    158,  // Totodile
-    328,  // Trapinch
-    252,  // Treecko
-    357,  // Tropius
-    157,  // Typhlosion
-    248,  // Tyranitar
-    236,  // Tyrogue
-    197,  // Umbreon
-    201,  // Unown
-    217,  // Ursaring
-    134,  // Vaporeon
-     49,  // Venomoth
-     48,  // Venonat
-      3,  // Venusaur
-    329,  // Vibrava
-     71,  // Victreebel
-    288,  // Vigoroth
-     45,  // Vileplume
-    313,  // Volbeat
-    100,  // Voltorb
-     37,  // Vulpix
-    320,  // Wailmer
-    321,  // Wailord
-    365,  // Walrein
-      8,  // Wartortle
-     13,  // Weedle
-     70,  // Weepinbell
-    110,  // Weezing
-    340,  // Whiscash
-    293,  // Whismur
-     40,  // Wigglytuff
-    278,  // Wingull
-    202,  // Wobbuffet
-    194,  // Wooper
-    265,  // Wurmple
-    360,  // Wynaut
-    178,  // Xatu
-    193,  // Yanma
-    335,  // Zangoose
-    145,  // Zapdos
-    263,  // Zigzagoon
-     41,  // Zubat
+	NATIONAL_DEX_OLD_UNOWN_B,
+	NATIONAL_DEX_OLD_UNOWN_C,
+	NATIONAL_DEX_OLD_UNOWN_D,
+	NATIONAL_DEX_OLD_UNOWN_E,
+	NATIONAL_DEX_OLD_UNOWN_F,
+	NATIONAL_DEX_OLD_UNOWN_G,
+	NATIONAL_DEX_OLD_UNOWN_H,
+	NATIONAL_DEX_OLD_UNOWN_I,
+	NATIONAL_DEX_OLD_UNOWN_J,
+	NATIONAL_DEX_OLD_UNOWN_K,
+	NATIONAL_DEX_OLD_UNOWN_L,
+	NATIONAL_DEX_OLD_UNOWN_M,
+	NATIONAL_DEX_OLD_UNOWN_N,
+	NATIONAL_DEX_OLD_UNOWN_O,
+	NATIONAL_DEX_OLD_UNOWN_P,
+	NATIONAL_DEX_OLD_UNOWN_Q,
+	NATIONAL_DEX_OLD_UNOWN_R,
+	NATIONAL_DEX_OLD_UNOWN_S,
+	NATIONAL_DEX_OLD_UNOWN_T,
+	NATIONAL_DEX_OLD_UNOWN_U,
+	NATIONAL_DEX_OLD_UNOWN_V,
+	NATIONAL_DEX_OLD_UNOWN_W,
+	NATIONAL_DEX_OLD_UNOWN_X,
+	NATIONAL_DEX_OLD_UNOWN_Y,
+	NATIONAL_DEX_OLD_UNOWN_Z,
+	NATIONAL_DEX_ABRA,  // Abra
```

Comment:
all of the comments that specify a pokemon can be removed now, since the constant makes that clear

Reply:
Should I leave the comments for the German names?

Reply:
no, those should also be removed.

Reply:
Oh, my mistake.  I think you're right--leave the german names.